lib/core/Canvas.js

```diff
@@ -581,7 +607,6 @@ Canvas.prototype.removeRootElement = function(rootElement) {
  * and returns the new root element.
  *
  * @param {Object|djs.model.Root} rootElement
- * @param {boolean} override
```

Comment:
Not sure if this is the way it should be done in JSDocs. I could not find information on how to mark it `@deprecated`

Reply:
Well, removing it from public API is one way to do it. Every linter would complain though.